### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by fatmahussein**
Add
-Contact form
-Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -226,6 +226,31 @@ <h5 id="frm9">Skills<i class="arrow2 right"></i></h5>
...
+    <div class="flex-container">
+
+    <div class="itemx">
+      <input type="text" id="name" name="user_name" placeholder="Yeremias" required >
+    </div>     
+    <div class="itemy">
+      <input type="email" id="email" name="email" placeholder="natajayanj14@gmail.com" required >
+    </div>       
+    <div class="itemz">
+      <textarea type="text" id="textarea" name="textarea" placeholder="Write your message here"  required></textarea>
```
Comment (brainconnect93) [inline index.html:245]: - [ ] I would kindly suggest you give the `name` and `message` input tags a max-length attribute. with this, you will follow the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) 👍 👍 👍 
- Name input (can be first name and last name field depending on the template used) has maxlength="30" attribute.
- Message input has maxlength="500" attribute.

File: index.html [hunk 2]
```diff
@@ -226,6 +226,31 @@ <h5 id="frm9">Skills<i class="arrow2 right"></i></h5>
...
+    </div>   
+
+    <div class="item" id="pro">
+    <a id="form-button" href="#" type="submit" class="button">Get It Touch</a>
```
Comment (brainconnect93) [inline index.html:249]: - [ ] I would kindly suggest you make the submit button be either an `input` element with type `submit` or a `button` element with type `submit` or with no `type` attribute (note that it defaults to a `submit` type when no `type` attribute is used.) 👍

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @fatmahussein, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -226,6 +226,31 @@ <h5 id="frm9">Skills<i class="arrow2 right"></i></h5>
...
+    <div class="itemz">
+      <textarea type="text" id="textarea" name="textarea" maxlength="500" placeholder="Write your message here"  required></textarea>
+    </div>   
+
+    <div class="item" id="pro">
+    <button id="form-button"  type="submit"> Get It Touch </button>
+   </div>
+  </div>
+  
+</form>
```
Comment (OyePriscilla) [inline index.html:253]: - [ ] Please I noticed that your form is not submitting data as expected, perhaps you might need to remove the div container and let each input be a direct child to the form element. Thanks. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @fatmahussein 😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (fatmahussein) [reply index.html:253]: Hi @OyePriscilla , My form is submitting
Screenshot (134)

Comment (fatmahussein) [reply index.html:253]: Please check again
Screenshot (135)

Comment (OyePriscilla) [reply index.html:253]: @fatmahussein Awesome, kindly submit for a re-review.

**Review by levy002 — APPROVED** [review]
## Hello @fatmahussein 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**